VideoWeb/VideoWeb/ClientApp/src/app/services/user-media-stream.service.ts

```diff
@@ -43,26 +45,40 @@ export class UserMediaStreamService {
         if (this.requestStream) {
             this.stopStream(this.requestStream);
         }
-
-        this.requestStream = await this.navigator.mediaDevices.getUserMedia(this.permissionConstraints);
-        return this.requestStream;
+        try {
+            this.requestStream = await this.navigator.mediaDevices.getUserMedia(this.permissionConstraints);
+            return this.requestStream;
+        } catch (error) {
+            this.logger.error('[UserMediaStreamService] - Could not get media stream', error);
```

Comment:
Please use a const for the logger prefix here, as discussed when the logger was demoed.

Reply:
Done